mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -104,7 +149,8 @@ def mask_cross_entropy(pred,
                        reduction='mean',
                        avg_factor=None,
                        class_weight=None,
-                       ignore_index=None):
+                       ignore_index=None,
```

Comment:
just follow above default value for `ignore_index`, and also revise the docstring for `ignore_index` of `mask_cross_entropy`
```suggestion
                       ignore_index=-100,
```